themes/default/content/registry/packages/aws/installation-configuration.md

```diff
@@ -208,3 +208,73 @@ Use `pulumi config set aws:<option>` or pass options to the [constructor of `new
+
+To do this, you will need to complete the following steps:
+
+### Configure OIDC between Pulumi and AWS
+
+Refer to the [Configuring OpenID Connect for AWS Guide](https://www.pulumi.com/docs/pulumi-cloud/oidc/aws/) for the step-by-step process on how to do this. Note that when adding your configuration to your environment file, you can also define your AWS region in the `environmentVariables` section:
+
+```yaml
+values:
+  aws:
+    login:
+      fn::open::aws-login:
+        oidc:
+          duration: 1h
+          roleArn: <your-oidc-iam-role-arn>
+          sessionName: pulumi-environments-session
+  environmentVariables:
+    AWS_ACCESS_KEY_ID: ${aws.login.accessKeyId}
+    AWS_SECRET_ACCESS_KEY: ${aws.login.secretAccessKey}
+    AWS_SESSION_TOKEN: ${aws.login.sessionToken}
+    AWS_REGION: <YOUR_AWS_REGION> # e.g.`ap-south-1`
+```
+
+### [Optional] Move Pulumi config to your ESC environment
+
+It was mentioned earlier in this guide that you can also set your AWS region as a Pulumi configuration value in your project's stack settings file (`Pulumi.<stack-name>.yaml`). In addition to this, there may be other values that you have defined in this file, and these values can also be centralized using Pulumi ESC. To [expose these values to Pulumi IaC](https://www.pulumi.com/docs/pulumi-cloud/esc/environments/#projecting-pulumi-config), you will need to add a second-level key called `pulumiConfig` and nest any desired values underneath it as shown below:
+
+```yaml
+values:
+  aws:
+    login:
+      fn::open::aws-login:
+        oidc:
+          duration: 1h
+          roleArn: <your-oidc-iam-role-arn>
+          sessionName: pulumi-environments-session
+  environmentVariables:
+    AWS_ACCESS_KEY_ID: ${aws.login.accessKeyId}
+    AWS_SECRET_ACCESS_KEY: ${aws.login.secretAccessKey}
+    AWS_SESSION_TOKEN: ${aws.login.sessionToken}
+  pulumiConfig: # exposes Pulumi config values to the Pulumi CLI
+    aws:region: <YOUR_AWS_REGION> # e.g.`ap-south-1`
+```
+
+Note that the configuration values under `pulumiConfig` can also be referenced directly from within your Pulumi program code. This is done using the same method to reference values from your project's stack settings file. You can see examples of how to do this in the [Accessing Configuration from Code](https://www.pulumi.com/docs/concepts/config/#code) section of the Pulumi documentation.
+
+### Import your environment
+
+The last step is to update your project's stack settings file (`Pulumi.<stack-name>.yaml`) to import your ESC environment as shown below:
+
+```yaml
+environment:
+  - <your-environment-name>
+```
+
+Make sure to replace `<your-environment-name>` with the name of the ESC environment you created in the previous steps.
+
+You can test that your configuration is working by running the `pulumi up` command. This will deploy your AWS resources using the dynamically generated credentials in your environment file.
```

Comment:
Might be better to suggest `preview`, since this won't deploy any resources, and will also fail if any of these values are missing:

```suggestion
You can test that your configuration is working by running `pulumi preview`. 
```